src/main/kotlin/no/nav/familie/oppdrag/iverksetting/OppdragSkjemaConstants.kt

```diff
@@ -17,7 +17,8 @@ class OppdragSkjemaConstants {
         const val ENHET = "8020"
         val ENHET_DATO_FOM: LocalDate = LocalDate.of(1900, 1, 1)
 
-        val FRADRAG_TILLEGG = TfradragTillegg.T
+        val FT_TILLEGG = TfradragTillegg.T
```

Comment:
Mangler litt hvorfor denne PR i PR-beskrivelsen, hva disse tingene er og evt konsekvens. For her snakker jeg altfor lite oppdragsk til å skjønne hva FT og FL er